LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadJoinAccept.cs

```diff
@@ -67,8 +67,8 @@ public LoRaPayloadJoinAccept(string netId, DevAddr devAddr, byte[] appNonce, byt
 
             var cfListLength = cfList == null ? 0 : cfList.Length;
             RawMessage = new byte[1 + 12 + cfListLength];
-            Mhdr = new Memory<byte>(RawMessage, 0, 1);
-            Array.Copy(new byte[] { 32 }, 0, RawMessage, 0, 1);
+            MHdr = new MacHeader(32);
```

Comment:
I think we should use the more meaningful constructor here:

```suggestion
            MHdr = new MacHeader(MacMessageType.JoinAccept);
```